src/Microsoft.OData.Edm/Schema/BadEntityType.cs

```diff
@@ -34,5 +35,7 @@ public bool HasStream
         {
             get { return false; }
         }
+
+        public IEnumerable<KeyValuePair<IEdmStructuralProperty, string>> DeclaredKeyAlias => throw new NotImplementedException();

```

Comment:
Would a dictionary mapping aliases to their properties (or property name) not be a better way to access aliases?